ds_judgements_public_ui/templates/includes/basic_search_form.html

```diff
@@ -11,7 +11,7 @@ <h2 class="search-component__header--sr-only">Search by keyword or neutral citat
     </form>
 
     <div class="search-component__more-options">
-      <a href="{% url 'structured_search' %}">More search options</a>
+      <a href="{% url 'structured_search' %}">Filter by court, date or person</a>
```

Comment:
Can we make this a translated string?

Reply:
Yep, you're absolutely right. Will do that now!

Reply:
Fixed!